features/facebookImport/src/model/analyses/report/report-metadata.js

```diff
@@ -16,10 +16,10 @@ export default class ReportMetadataAnalysis extends ReportAnalysis {
         };
     }
 
-    async analyze({ size, zipFile, facebookAccount }) {
+    async analyze({ size, zipFile, facebookAccount, pod }) {
```

Comment:
Definitely better than waiting for the window, and we can use mocks for testing. Still, it's an implementation-dependent feature that we could surely live without...

Reply:
Right.

Reply:
Now most analyses either use the zip file to work directly on the raw list of files or on the Facebook account. I think just the metadata analysis needs the pod and the size. We can definitely remove at least the file data easily from the list of parameters.